keyboards/lily58/keymaps/default/config.h

```diff
@@ -29,6 +29,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define QUICK_TAP_TERM 0
 #define TAPPING_TERM 100
 
+#define FORCE_NKRO
```

Comment:
I'm not a fan of forcing NKRO to be enabled, as it can cause issues for some systems.  There is a keycode for toggling the status, and that is saved in memory.

Reply:
didn't know that, I removed it